core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2555,6 +2555,15 @@ private[spark] object Utils extends Logging {
       sparkJars.map(_.split(",")).map(_.filter(_.nonEmpty)).toSeq.flatten
     }
   }
+
+  private[util] val REDACTION_REPLACEMENT_TEXT = "*********(redacted)"
+  def redact(conf: SparkConf)(kv: (String, String)): (String, String) = {
```

Comment:
I mean the `redact` method should take 2 parameters, the config and the list of things to be redacted, instead of using currying for the second parameter.